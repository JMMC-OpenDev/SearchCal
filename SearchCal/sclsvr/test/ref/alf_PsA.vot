version https://git-lfs.github.com/spec/v1
oid sha256:70f642a8c001af7f4a18fa704e0a5947b2bc82c1804b8fd686086b9381ca788d
size 130717
