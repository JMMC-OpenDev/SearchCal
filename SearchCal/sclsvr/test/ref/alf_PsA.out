version https://git-lfs.github.com/spec/v1
oid sha256:2f415ecebdb83a1fea25a39dba78451a251b4504411a070af48248ff44801c45
size 195100
